src/lqcut.jl

```diff
@@ -88,97 +95,111 @@ function lq_drift_time_correction(
+    par = mvalue(result_µ.par)
+    pol_fit_func = report_µ.f_fit
 
     #property function for drift time correction
-    lq_class_func = "(lq / ($(e_expression))$(e_unit)^-1) - ($(parameters[2]) * (qdrift / ($(e_expression))$(e_unit)^-1) + $(parameters[1]))"  # removes the units to get unitless lq classifier
+    lq_class_func = "$lq_e_corr_expression - " * join(["$(par[i]) * $dt_eff_expression^$(i-1)" for i in eachindex(par)], " - ")
     lq_class_func_generic = "lq / e  - (slope * qdrift / e + y_inter)"
 
     #create result and report
     result = (
     func = lq_class_func,
     func_generic = lq_class_func_generic,
+    fit_result = result_µ,
+    box_constraints = box,
     )
 
     report = (
-    lq_prehist = lq_prehist, 
     lq_report = lq_report, 
-    drift_prehist = drift_prehist, 
     drift_report = drift_report,
     lq_box = box,
-    drift_time_func = drift_time_func,
-    DEP_left = DEP_left,
-    DEP_right = DEP_right,
+    drift_time_func = pol_fit_func,
+    dep_left = dep_left,
+    dep_right = dep_right,
     )
     
 
     return result, report
 end
-export lq_drift_time_correction
+export lq_ctc_correction
 
 """
-    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{Float64}; cut_sigma::Float64=3.0, truncation_sigma::Float64=2.0)
+    lq_cut(dep_µ::Unitful.Energy, dep_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, dep_sideband_sigma::Float64=4.5, cut_truncation_sigma::Float64=2.0) )
+
+    Evaluates the cutoff value for the LQ cut. The function performs a binned gaussian fit on the sidebandsubtracted LQ histogram and evaluates the cutoff value difined at 3σ of the fit.
+
+# Arguments
+    * `dep_µ`: Mean of the DEP peak
+    * `dep_σ`: Standard deviation of the DEP peak
+    * `e_cal`: Energy
+    * `lq_classifier`: LQ classifier
+
+# Keywords
+    * `cut_sigma`: Number of standard deviations used to define the final cutoff value
+    * `dep_sideband_sigma`: Number of standard deviations used to define the sideband edges
+    * `cut_truncation_sigma`: Number of standard deviations used for the precut of sideband subtracted histogram
 
-Evaluates the cutoff value for the LQ cut. The function performs a binned gaussian fit on the sidebandsubtracted LQ histogram and evaluates the cutoff value difined at 3σ of the fit.
 # Returns
     * `result`: NamedTuple of the cutoff value
     * `report`: NamedTuple of the fit result, fit report and temporary histograms
+
 """
-function LQ_cut(
-    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, truncation_sigma::Float64=2.0)
+function lq_cut(
+    dep_µ::Unitful.Energy, dep_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, dep_sideband_sigma::Float64=4.5, cut_truncation_sigma::Float64=3.5)
```

Comment:
A quick comment why you changed the default value of the `cut_truncation_sigma` from `2.0` to `3.5`?